podman_hpc/podman_hpc.py

```diff
@@ -68,7 +64,7 @@ def get_default_containers_conf(self):
                     'seccomp_profile': 'unconfined',
                     'runtime': self.runtime
                   }
-               }
+                }
```

Comment:
I always like to `except Exception as e` and print e, just for easier debugging

Reply:
This is a fallback for cases where getlogin doesn't work.  So in that case we would just get it from the env.  I could try to catch the exact error getlogin might throw.

Reply:
Made it just catch OSError